topics/transcriptomics/tutorials/scrna_preprocessing/tutorial.md

```diff
@@ -0,0 +1,1385 @@
+>
+> 1. Are UMIs specific to genes? i.e. Can the same UMI map to different genes?
+> 2. Can the same UMI map to different mRNA molecules of the same gene?
+> 
+> > ### {% icon solution %} Solution
+> >
+> 1. No. The same UMI can tag transcripts of different genes. UMIs are not universal tags, they are just 'added randomness' that help reduce amplification bias -- not unique to any particular gene.
+> > 2. Yes, UMIs are not precise but operate probabilistically. In most cases, two transcripts of the same gene will be tagged by different UMIs. In rarer (but still prevalent) cases, the same UMI will capture different transcripts of the same gene.
+> >  * One helpful way to think about how quantification is performed is to observe the following heirarchy of data `Cell Barcode → Gene → UMI`
+> >
+> >   e.g.
+> > 
+> >  | BC:Cell | BC:UMI | Maps to Gene |
+> >  |------|-----|------|
+> >  | AAAT | TCA | Slx1 |
+> >  | AAAT | GTG | Slx2 |
+> >  | AAAT | TCA | Gh13 |
+> >  | TTAA | TCA | Slx1 |
+> >  | TTAA | CCC | Atp3 |
+> >
+> > If UMIs were unique to a gene, then the `TCA` UMI barcode would not have reads that map to both *Slx1* and *Gh13* in the same cell (`AAAT`).
+> >
+> {: .solution}
+>
+{: .question}
+
+### Barcoding Format
+
+We now know the role of UMIs and cell barcodes, but how do we handle them in the analysis? Let us look at 4 example sequences in our paired-end FASTQ data.
+
+> ### {% icon hands_on %} Hands-on: Preparing the Data
+>
+>    1. Create a new history and rename it (*e.g.* 'Inspecting FastQ Files in scRNA batch data')
+>    1. Import the following files from [`Zenodo`](https://zenodo.org/record/2554612) or from the data library (ask your instructor)
+>    ```
+>    https://zenodo.org/record/2554612/files/test_barcodes_celseq2_R1.fastq.gz?download=1
+>    https://zenodo.org/record/2554612/files/test_barcodes_celseq2_R2.fastq.gz?download=1
+>    ```
+>    
+>    {% include snippets/import_via_link.md %}
+>    
+>    
+>    1. Build a *Dataset pair* for the two FASTQ files
+>       - Click the *Operations on multiple datasets* check box at the top of the history panel
+>       - Check the two boxes next to the R1 and R2 scRNA FASTQ samples
+>       - Click *For all selected...* and choose *Build dataset pair*
+>       - Ensure that the forward read is the `R1` sample, and the reverse read is the `R2` sample.
+>          - Click 'Swap' otherwise.
+>       - Set the name of the pair
+>
+>    1. Generate a list of reads to filter by creating a plain tabular file containing the following read names:
+>        ```
+>        J00182:75:HTKJNBBXX:2:1114:12469:11073
+>        J00182:75:HTKJNBBXX:2:2222:13301:35690
+>        J00182:75:HTKJNBBXX:2:1203:25022:13763
+>        J00182:75:HTKJNBBXX:2:1115:8501:46961
+>        ```
+>    1. Set the datatype of the file as **tabular**
+>    1. Click the *View all histories* icon
+>    1. Drag the FASTQ collection from your previous history into your new history
```

Comment:
which previous history? Is this not about the celseq2 files that got just imported above? I'm confused.